open-api/rest-catalog-open-api.yaml

```diff
@@ -1581,6 +1607,17 @@ components:
         type: string
       example: [ "accounting", "tax" ]
 
+    PageToken:
+      description:
+        An opaque token which allows clients to make use of pagination for a list API (e.g. ListTables) by signaling to the service
+        that they would prefer the response to be paginated.
+
+        Clients will initiate the request by sending an empty pageToken e.g. GET /tables?pageToken or /tables?pageToken=
```

Comment:
```suggestion
        Clients will initiate the request by sending an empty page-token, e.g. GET /tables?page-token or /tables?page-token=
```

Reply:
I think query parameters need to be in camel case, for example `purgeRequested`

Reply:
@jackye1995 is correct, the query params can be camel case. https://github.com/apache/iceberg/blob/main/open-api/rest-catalog-open-api.yaml#L776

Reply:
yeah sorry I was confused whether we're using pageToken vs page-token